dockstore-launcher/src/main/java/io/github/collaboratory/LauncherCWL.java

```diff
@@ -425,7 +425,24 @@ private String setupDirectories() {
 
     private Map<String, Object> runCWLCommand(String cwlFile, String jsonSettings, String outputDir, String workingDir) {
         String[] s = {"cwltool","--enable-dev","--non-strict","--enable-net","--outdir", outputDir, "--tmpdir-prefix", workingDir, cwlFile, jsonSettings};
-        final String joined = Joiner.on(" ").join(Arrays.asList(s));
+
```

Comment:
Small nitpick, I think this could be cleaner by not using both `s` and `newCommand`
Also, you should be able to clean up the loop with a lambda https://stackoverflow.com/questions/28319064/java-8-best-way-to-transform-a-list-map-or-foreach